jdaviz/configs/default/plugins/plot_options/plot_options.vue

```diff
@@ -68,6 +68,14 @@
       </glue-state-sync-wrapper>
     </div>
 
+    <div v-if="image_color_mode_value === 'One color per layer'">
+      <v-row justify="end">
+        <j-tooltip tooltipcontent="Apply preset RGB colors, scaling, and opacities to visible layers">
+          <v-btn color="primary" text @click="apply_RGB_presets">Assign Presets</v-btn>

```

Comment:
This case isn't really covered in our style-guide, but I _think_ since this is applying changes back to the viewer, we want `color="accent"`.  We should add this to the list of things to ask @Jenneh when doing final UI tweaks though!

Reply:
I went ahead and changed it.